spec/support/utilities_spec.rb

```diff
@@ -1,3 +1,16 @@
-require 'support/sign_in_helper_spec'
```

Comment:
Can you find out what Capybara expects from a button element? Eg a certain class of html element? What makes the function click_button working? Probably this Element does not appear in the test environment (then try to make it appear). Or: is this button or the I18n-key still named aus the test suggests?